tests/plenary/ui/mappings_spec.lua

```diff
@@ -1568,4 +1568,28 @@ describe('Mappings', function()
     vim.cmd('norm ,o,a\r')
     assert.are.same('* [#A] Test orgmode', vim.fn.getline(1))
   end)
+
+  it('should refile to headline that matches name exactly', function()
+    local destination_file = helpers.load_file_content({
+      '* foobar',
+      '* baz',
+      '* foo',
+    })
+
+    local source_file = helpers.load_file_content({
+      '* to be refiled',
+      '* not to be refiled',
+    })
+
+    vim.fn.cursor(1, 1)
+    vim.cmd([[norm ,or]])
+    -- TODO: this is wrong, not a command. how to do the actual refiling?
```

Comment:
The function for this test is likely 
```lua
Capture:refile_to_headline()
```

Reply:
This works because Capture "class" doesn't have a lot of it's own internal state, but we should avoid calling these things directly. Capture is instantiated on the main Org class.
I just pushed a small change that can be used here like this:
```lua
require('orgmode').instance().capture:refile_to_headline()
```